What's New 8.4 updates (#2126)
elastic/observability-docs: docs/en/observability/whats-new.asciidoc

Here are the highlights of what's new and improved in {minor-version}.

Other versions:
{observability-guide-all}/8.3/whats-new.html[8.3] |
{observability-guide-all}/8.2/whats-new.html[8.2] |
{observability-guide-all}/8.1/whats-new.html[8.1] |
{observability-guide-all}/8.0/whats-new.html[8.0] |
{observability-guide-all}/7.17/whats-new.html[7.17] |

// tag::whats-new[]

// What's new content goes in here. Don't uncomment or remove the tags surrounding this content :)

[discrete]
== New Synthetics App enhancements for public beta

As we progress towards delivering an all new Synthetic Monitoring capability
within Elastic Observability, we’re excited to announce two big new enhancements
for our 8.4 stack release:

- Private testing locations
- Open access to the managed testing service for all Elastic Cloud users

With private testing locations you can now configure a testing location on

> self-managed infrastructure. This means that you can test applications that
> aren’t publically available on the internet or from locations where we may not

Reviewer: ```suggestion
self-managed infrastructure. This means you can test applications that
aren’t publicly available on the internet or from locations where we may not
```

Author: omg, my head is indeed fried lol

yet have a point of presence.

A big part of our new Synthetics app is the ability to run synthetic monitoring
tests on our new global network of managed testing locations. With the 8.4

> release, we’ll no longer be requiring users to request access. If you want to
> use our managed testing infrastructure, simply open the {uptime-app} and click

Reviewer: I don't like "simply" but you can leave it if you do.

```suggestion
release, we’ll no longer be requiring users to request access. To
use our managed testing infrastructure, open the {uptime-app} and click
```

*Monitor Management*. There is a fair usage cap in place but all test executions

> are free. We’d love to get you involved and get your feedback.

Reviewer 2: Might be worth telling users how to submit feedback.

Author: I did think similar...will need to get something concrete via Drew though, might be a late addition

Reviewer 2: Yah definitely not worth delaying the merge. Just something to add later.

[discrete]
== Observability alerting and rules views now generally available (GA)

Alerting and rule management for Observability-specific alerts and rules have
been offered as a technical preview since 7.16. We have made significant
improvements since then and we now have robust, feature-rich
capabilities that are production ready.

[role="screenshot"]
image::images/observability-alerts-overview.png[Observability Alerts view]

Working alongside Cases functionality for incident management, Observability

> Alerts and Rules views provide you with an easy way to access and manage

Reviewer: ```suggestion
Alerts and Rules views provide an easy way to access and manage
```

Observability-specific rules, alerts, and cases within the Observability UI.
With comprehensive filtering and search support, you can quickly view or edit
alerts or rules of interest.

[role="screenshot"]
image::images/observability-rules.png[Observability Rules view]

The Rule Detail view offers rule creators and
administrators a complete history of rule executions and seamless access to
generated alerts for effective management. In 8.4 the Rule Details view has been
further enhanced to include an at-a-glance alert summary that shows the status
and timeline for triggered alerts, along with a number of other UI improvements,
including bulk action support.

[role="screenshot"]
image::images/rules-cpu-exhaustion.png[Observability Rule detail]

> For more information on Observability alerting, see <<create-alerts>>.

Reviewer: My mistake. This is reused in the stack docs so this needs to be a cross-doc ref

```suggestion
For more information on Observability alerting, see {observability-guide}/create-alerts.html[Create alerts].
```

> For more information on Observability alerting, see {observability-guide}/create-alerts.html[Create alerts].

Reviewer 2: ```suggestion
For more information on Observability alerting, refer to {observability-guide}/create-alerts.html[Create alerts].
```

Author: Ah, are we now favouring "refer" to "see"? I'm not sure I agree either, seems a bit less plain English / slightly puritanical, but I'll follow the guidelines.

Reviewer 2: Yah I totally don't agree...because I have a close friend who is blind, and he uses "see" all the time. :-) 

Here are the accessibility guidelines: https://docs.elastic.dev/content-design/accessibility

I also don't agree to using "check" because the word has multiple meanings and can get mistranslated (thinking forward to someday when that matters....)

> For more information on Observability alerting, see the https://www.elastic.co/guide/en/observability/current/create-alerts.html[Alerts documentation]

Reviewer: ```suggestion
For more information on Observability alerting, see <<create-alerts>>
```

[discrete]
== Improved RCA flows for erroneous services

With two new _Top 5_ views introduced for version 8.4, you can quickly identify
the impact of application errors and reduce time spent identifying and tracking
errors.

> The *Top 5 errors* view is available from the APM Transaction details page:

Reviewer 2: We're a bit inconsistent with highlighting in this topic (sometimes the view name is bold, sometimes it is not).

Author: Yeah, agreed - I'm also a little confused by what is specifically a view or not when the screenshot doesn't seem to match the supplied text e.g. Rule Detail view, it's a bit of a fudge in places

[role="screenshot"]
image::images/apm-services-top-5-errors.png[Top 5 errors in APM Services]

The *Top 5 affected transactions* view is available from the APM Error Group
details pages:

[role="screenshot"]
image::images/apm-services-top-5-transactions.png[Top 5 affected transactions in APM Services]

[discrete]
== Contextual view of infrastructure metrics for APM Services (Beta)

Bringing together metric and trace data provides powerful insights to help you
understand application performance and identify the source of any issues.

> Available as a beta feature in 8.4, Infrastructure metrics for container, pod,
> and host are available in the context of individual services. You can use the new

Reviewer: ```suggestion
Available as a beta feature in 8.4, Infrastructure metrics for containers, pods,
and hosts are available in the context of individual services. You can use the new
```

Infrastructure tab in the APM Service Overview to get a summary view of the
infrastructure supporting a service, with the ability to drill down into metrics
for additional details.

[role="screenshot"]
image::images/apm-services-infrastructure.png[Infrastructure metrics for APM Services]

[discrete]
== Native onboarding flow for Open Telemetry instrumentation

With Elastic APM already natively supporting the OpenTelemetry protocol,
instrumentation instructions have been added to Elastic {apm-agent} onboarding to
simplify and streamline the use of OpenTelemetry agents with Elastic APM.

[role="screenshot"]
image::images/open-telemetry-apm-agent.png[APM Agent onboarding for OpenTelemetry]

In addition, APM Server has now added support for ingestion of OpenTelemetry
summary metrics (sum and count), including those created by Prometheus exporters.

> For more information, see the {apm-guide-ref}/open-telemetry.html[OpenTelemetry documentation]

Reviewer 2: ```suggestion
For more information, refer to the {apm-guide-ref}/open-telemetry.html[OpenTelemetry documentation]
```
